docs/native-components-android.md

```diff
@@ -273,8 +488,121 @@ public class MyFragment extends Fragment {
 }
 ```
 
+</TabItem>
+</Tabs>
+
 ### 3. Create the `ViewManager` subclass
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin title="MyViewManager.kt"
+// replace with your package
+package com.mypackage
+
+import android.view.Choreographer
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.fragment.app.FragmentActivity
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReadableArray
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.ViewGroupManager
+import com.facebook.react.uimanager.annotations.ReactPropGroup
+import kotlin.properties.Delegates
+
+class MyViewManager(
+    private val reactContext: ReactApplicationContext
+) : ViewGroupManager<FrameLayout>() {
+  private var propWidth by Delegates.notNull<Int>()
+  private var propHeight by Delegates.notNull<Int>()
```

Comment:
Not sure about this. Should they just be `var Int?`?

Reply:
Can do. Delegates are quite nice because we delegate the null-handling (accessing them before they're initialized will throw an exception with a nice message) and we don't have to spam `requireNotNull` each and every time we access the variables. In this case we just access them in a single function so it doesn't offer a lot of benefit though.

And for the purposes of this tutorial it might be too confusing / advanced so yes, let's make them nullable and use `requireNotNull`.